theatre/studio/src/store/stateEditors.ts

```diff
@@ -245,6 +245,42 @@ namespace stateEditors {
                   ])
                 }
               }
+
+              export function replaceMarker(
+                p: SheetAddress,
+                marker: {time: number},
```

Comment:
as a convention, all fns in `stateEditors` take a single argument. Reason is, all arguments will have a unique prop name, which should make it a bit easier to later these state editors to a concurrent editing setup.